src/Expander.hs

```diff
@@ -1328,24 +1329,24 @@ addApp ctor (Syntax (Stx scs loc _)) args =
     app = Syntax (Stx scs loc (Id "#%app"))
 
 problemContext :: MacroDest -> MacroContext
-problemContext (DeclDest _ _ _) = DeclarationCtx
+problemContext (ModBodyDest _ _ _) = DeclarationCtx
 problemContext (TypeDest _) = TypeCtx
 problemContext (ExprDest _ _) = ExpressionCtx
 problemContext (PatternDest _ _ _) = PatternCaseCtx
 
-requireDecl :: Syntax -> MacroDest -> Expand (Scope, DeclPtr, DeclValidityPtr)
-requireDecl _ (DeclDest dest sc ph) = return (sc, dest, ph)
-requireDecl stx other =
+requireDeclarationCtx :: Syntax -> MacroDest -> Expand (Scope, ModBodyPtr, DeclValidityPtr)
+requireDeclarationCtx _ (ModBodyDest dest sc ph) = return (sc, dest, ph)
+requireDeclarationCtx stx other =
```

Comment:
`requireModBody` sounded wrong, so I renamed this family of function to match the `MacroContext` constructor names.

Reply:
It would now be `requireDeclTree` instead of `requireModBody`, which doesn't sound as wrong, so I could revert this if you want. I like the new names a bit better though.

Reply:
This sounds like an improvement to me!